src/hist/basehist.py

```diff
@@ -284,34 +284,24 @@ def plot1d(
         if self.ndim != 1:
             raise TypeError("Only 1D-histogram has plot1d")
 
-        """
-        Default Figure: construct the figure and axes
-        """
+        # Default Figure: construct the figure and axes
```

Comment:
Let's use normal comment blocks, and only use strings at the top of functions, where they turn into docstrings.